### DISCUSSION TITLE
I love the new GUI changes

### DISCUSSION THREAD

**Discussion by LockMan007** [obsproject/obs-studio#11316] [General]
I love the new GUI changes and performance optimizations.
Thanks for the updates and also thanks for the new video format for MP4.
I now love OBS.

**Comment by look997** [comment]
@LockMan007 What changes? Show screenshots and describe them.

**Reply by LockMan007** [reply]
It changed a few versions ago, not sure when, I put off updating for a while and when I did, they modernized the way the main GUI looks and added quicker access to making some common changes. It's not really important, just saying I like the changes and LOVE the new MP4 container format!

**Comment by Panda-Bro4you** [comment]
Another thing I really appreciate is the improved performance and smoother navigation throughout the application. Startup time seems faster, menus are more responsive, and the overall workflow feels more streamlined. I also like the better organization of options, which makes advanced features easier to find without digging through multiple menus.

The new customization options are a nice touch as well, allowing users to personalize the layout and behavior to better suit their workflow. Stability improvements and bug fixes also seem noticeable compared to earlier versions.

And of course, I absolutely LOVE the new MP4 container format support! It’s much more flexible and compatible with modern media players, making file handling and sharing a lot easier. Overall, the updates really improved usability while keeping the core functionality that made the software great in the first place.
